### PR TITLE
131 doc updates

### PR DESCRIPTION
**Pull request by phlptp**
Fix #467 and add some additional support .md file for Github
move a few files to the .github folder

add a test for #441, but haven't replicated the issues.

### CODE REVIEW

**Comment by phlptp** [comment]
@nightlark  any idea what is going on with the push branch,  looks like something is wrong with the boost build, but I am not sure why?  I tried deleting the cache but that didn't do anything.

**Comment by phlptp** [comment]
@kdheepak  can you take a look at this PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This section guides you through submitting a feature request, or enhancement for
 #### Before Submitting An Enhancement Suggestion
 
 * check the issue list for any similar issues
-* take a look at the [RoadMap](ROADMAP.md) or projects to see if the feature is already planned
+* take a look at the [RoadMap](../ROADMAP.md) or projects to see if the feature is already planned
```
Comment (kdheepak) [inline CONTRIBUTING.md]: Why is this going back to the previous directory? Shouldn't it be `./ROADMAP.md`?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
+
+Fixes #
```
Comment (kdheepak) [inline .github/PULL_REQUEST_TEMPLATE.md]: We should put directions for users in a markdown comment, and put only the template of the PR we want in markdown.

```markdown
<!-- 

Please review guidelines

-->

- [ ] Have you reviewed the guidelines?
 
```

 Thoughts?

File: CONTRIBUTING.md (hunk 1 above)
Comment (phlptp) [reply CONTRIBUTING.md]: That is a mistake.  I at one point moved this file to the .github folder, but then started looking through what other projects have done and digging a little and it was strongly suggested that the CONTRIBUTING.md should be in the root folder to be easily found, outside of github.  So I moved it back, and didn't fix all the links.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (phlptp) [reply .github/PULL_REQUEST_TEMPLATE.md]: I am not sure I would want a checklist for users to do in the templates.  Do you think that is a good idea?  Also I am not sure the link will work if we put that in a comment?

Comment (kdheepak) [reply .github/PULL_REQUEST_TEMPLATE.md]: The link won't work, but I think it's fine to provide the high level instructions there, e.g. following.

```
<!-- 
Please make sure you fill the following sections. If this PR fixes an issue, please tag the issue number in the first section.

e.g.

This fixes issue #123
-->

- [ ] I have reviewed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] I agree to license my contributions under the same license as in this repository
- [ ] I have verified that the tests pass

# Description

<!-- Describe your Pull Request here -->

# Proposed changes

<!-- Describe the proposed changes of Pull Request here -->

-
- 
-

```

The checklist is useful for list of things people should consider before submitting the PR. What do you think of the above example?

Comment (phlptp) [reply .github/PULL_REQUEST_TEMPLATE.md]: I am OK with that,  if we put the link in comments it should be the global link, not a relative one though.  @kdheepak Can you add a commit with the changes?,  then we can merge this.

**Comment by phlptp** [comment]
@kdheepak, @nightlark Is there any other changes we want to make for a 1.3.1 release?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (kdheepak) [reply .github/PULL_REQUEST_TEMPLATE.md]: I can make the change later tonight.

**Comment by kdheepak** [comment]
I think we should be good! I want to make more documentation changes, to write out and describe how to use helics in more detail. But I can push those changes to `ghpages` even without updating master on github.

**Comment by phlptp** [comment]
@kdheepak  were you going to fix #446  yet or is that part of the documentation?

**Pull request merged**